waspc/cli/src/Wasp/Cli/Command/Db/Start.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Wasp.Cli.Command.Db.Start
+  ( start,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (Abs, Dir, Dir', Path', Rel, fromAbsDir, reldir, (</>))
+import System.Process (callCommand)
+import qualified Wasp.AppSpec as AS
+import qualified Wasp.AppSpec.App as AS.App
+import qualified Wasp.AppSpec.App.Db as AS.App.Db
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Common
+  ( findWaspProjectRootDirFromCwd,
+    parseWaspFileC,
+  )
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import Wasp.Cli.Common (WaspProjectDir, dotWaspDirInWaspProjectDir)
+import qualified Wasp.Db.Postgres as P
+import qualified Wasp.Message as Msg
+
+start :: Command ()
```

Comment:
As we talked IRL, I would maybe add a bit of explanation when the PostgreSQL db is started,

```
You are using a PostgreSQL database system in the Wasp file, so we started a local dev PostgreSQL instance for you to use.

You don't need to set up anything in the .env.server file, but if you are looking for the connection string for some database tool, here it is:

postgresql://...
```

Reply:
If have a `DATABASE_URL` set up, my app will use it. It would be great if we could check for that and output a message to the user.

Reply:
Yeah, I agree. I got confused for a second when I ran `wasp db start` and then started the app, but had my .server.env file there and it was using my regular PG DB instead of the Wasp/Docker version.

Reply:
@infomiho this is a cool idea, I will add it!

@infomiho & @shayneczyzewski yes I would like to add that check for existing DATABASE_URL, I have actually have it there under the TODOs, although I put it under more advanced TODOs since it is mostly important for old users + it requires a bit of effort to implement (parsing .env.server) so I am considering postponing it for a later PR.